pyproject.toml

```diff
@@ -52,24 +52,24 @@ protobuf = { version = "^4.21", optional = true }
 docs = ["sphinx", "sphinx_rtd_theme"]
 grpc = ["grpcio", "protobuf"]
 
-# Test
-[tool.poetry.group.dev.dependencies]
-pytest = ">=7.2"
-pytest-cov = ">=4.0"
-pytest-mock = ">=3.0"
-pykka = "^3.0"
-tox = "^3.24"
-ni-python-styleguide = ">=0.4.1"
-sphinx_rtd_theme = "1.0"
```

Comment:
Do we need to lock to `sphinx_rtd_theme` equals exactly 1.0?

Reply:
No, that's not a requirement. This looks like bad translation to poetry by me.